tests/config.spec.ts

```diff
@@ -16,29 +16,37 @@
 
 import fs from 'node:fs';
 
-import { Config } from '../config.js';
 import { test, expect } from './fixtures.js';
 
 test('config user data dir', async ({ startClient, localOutputPath, server }) => {
   server.setContent('/', `
     <title>Title</title>
     <body>Hello, world!</body>
   `, 'text/html');
-
-  const config: Config = {
-    browser: {
-      userDataDir: localOutputPath('user-data-dir'),
-    },
-  };
-  const configPath = localOutputPath('config.json');
-  await fs.promises.writeFile(configPath, JSON.stringify(config, null, 2));
-
-  const client = await startClient({ args: ['--config', configPath] });
+  const userDataDir = localOutputPath('user-data-dir');
```

Comment:
What was wrong with this test?

Reply:
nothing, I saw an opportunity to refactor - but it seems I don't understand what I was doing